tests/refinements/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2565,7 +2565,7 @@ let tests = [
     if (x.kind === null.toString()) {
     }
     // error, method on null
-    if ({ kind: 1 }.kind === null.toString()) {
+    if (({ kind: 1 }).kind === null.toString()) {
```

Comment:
This seems to add unnecessary brackets. Maybe we should add a method to see if the current scope expects an `Expression`, and avoid unnecessary brackets?